### ISSUE TITLE
Group Page needs fine tuning

### ISSUE THREAD

**Issue by satu0king** [CircuitVerse/CircuitVerse#1767]
**Describe the bug**
The time column takes lot of space. And because of that, the title column is going into 2 lines. 

![image](https://user-images.githubusercontent.com/2092958/94961173-c6a1af80-0511-11eb-8f15-7e04f3096050.png)

Not sure what can be done. I suppose we can remove time zone detail and GMT?

@nitin10s @aayush-05 @Shivansh2407 @sal2701 any inputs here?  

**Are you working on this issue? (Yes/No)**
No

**Comment by aayush-05** [comment]
Umm, not sure if we should remove the timezone information altogether since now maybe with remote classes students and teachers could be residing in different ones. We can surely remove the "GMT+0530" and even the seconds information since it is always gonna be 00. Other thing we can do is maybe via some string manipulation we can reduce "India Standard Time" to IST (this should be achievable since we have all timezones as a three-word string afaik)

**Comment by satu0king** [comment]
@aayush-05 I think if we set the bootstrap class attribute appropriately, it will work?

**Comment by aayush-05** [comment]
Yes, just required to increase column span for Name column and decrease the same for Deadline one.

**Comment by satu0king** [comment]
Also graded column can be merged with name? 
Like 

7 segment display
Graded(letter)

**Comment by sal2701** [comment]
I feel we can try few things
- Round off the time, Instead of 5 days 15 hours 30 minutes, just display 5 days 15 hours. Once the deadline approaches, make it hours, minutes. Basically display just 2 units of time.
- Regarding the timezone, we could display in local timezone. As using the "Time" and "Time Left" user can easily figure out the deadline. And if necessary do something like Codeforces, add a link like [this](https://www.timeanddate.com/worldclock/fixedtime.html?day=30&month=9&year=2020&hour=17&min=35&sec=0&p1=166) just to be sure.

**Comment by satu0king** [comment]
I like the rounding off idea. Remaining time in days + hours, then hours + minutes as it becomes less than 1 day. Also we can consider not displaying the date in words and rather 03-02-2020

**Comment by aayush-05** [comment]
Merging graded column would save up a lot of space. Personally I don't agree with removing the timezone text altogether; add in some work for user to figure out

**Comment by tachyons** [comment]
Timezone information is important as we are not showing the time based on user's time zone yet. So showing IST time without specifying IST mark will be misleading

**Comment by tachyons** [comment]
We need to allow students to set their timezone

**Comment by aayush-05** [comment]
We are changing time based on user's timezone (as provided by browser). So the user can input time in his timezone while creating an assignment. It will be first changed to UTC and then saved to dB. Further, whenever it will be shown to other users; the time in UTC will be fetched from dB, changed according to user's timezone and then displayed.
Not all users are seeing time in IST, rather in timezone specified by their browsers. Refer [this](https://github.com/CircuitVerse/CircuitVerse/blob/4b249979c531c757d6faee18a53f4c2e9802286b/app/views/assignments/_form.html.erb#L53) and [this](https://github.com/CircuitVerse/CircuitVerse/blob/4b249979c531c757d6faee18a53f4c2e9802286b/app/views/assignments/show.html.erb#L164)

**Comment by satu0king** [comment]
Yes @aayush-05 so does it mean we can remove the timezone?

**Comment by aayush-05** [comment]
Not exactly, it clarifies the user that the time shown is in his local timezone. Minifying something like GMT+0530 (Indian Standard Time) to IST would do along with decreasing that column span
